package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@dellstorage/clarity-react",
-    "version": "1.2.5",
```

Comment:
Why did we jump to 1.2.7?  Why not 1.2.6.
So local Bumping pushed it ?

Reply:
Checking the package.json in ObjectScale Portal, I see that version attached is 1.2.6, but version in main is 1.2.5. Hence updated version to +1 as I believe one version got skipped.

Is it safe to keep as 1.2.6 instead?

Reply:
I remember the issue with publishing now.  So normally when we run the git job to publish to npm, it will bump the version as well, but it's not committing back the version on our change.  

So even if the repo shows 1.2.5, the actual published version is 1.2.6
In your case, the repo shows 1.2.7 but, when you publish, it will publish 1.2.8

Just after publishing, please raise another PR to bump the version to 1.2.8 for future.